test/checkpoint.test.js

```diff
@@ -1,20 +1,21 @@
 var async = require('async');
 var loopback = require('../');
-
-// create a unique Checkpoint model
-var Checkpoint = loopback.Checkpoint.extend('TestCheckpoint');
-
-var memory = loopback.createDataSource({
-  connector: loopback.Memory
-});
-Checkpoint.attachTo(memory);
+var expect = require('chai').expect;
 
 describe('Checkpoint', function() {
-  describe('current()', function() {
+  var Checkpoint = loopback.Checkpoint.extend('TestCheckpoint');
```

Comment:
Would you mind moving this line back to L5 (see above), to keep the diff smaller?

```js
var async = require('async');
var loopback = require('../');
var expect = require('chai').expect;

var Checkpoint = loopback.Checkpoint.extend('TestCheckpoint');

describe('Checkpoint', function() {
  // ...
});
```